examples/repeated-measurements.jmd

```diff
@@ -0,0 +1,118 @@
+---
+title: "Soss Example: Industrial Quality Control"
+author: Chad Scherrer
+weave_options:
+    line_width: 1500
+    dpi: 200 
+---
+
+This is a very simplified example of how Soss might be used for quality control of an industrial process. To make things concrete, let's say we observe data like this:
+
+```julia; echo=false
+using NamedArrays 
+
+using Soss
+m = @model r,s begin
+    p_bad ~ Beta(1,3) |> iid(s)
+    bad ~ For(s) do j
+            Bernoulli(p_bad[j])
+        end
+    
+    fpr ~ Beta(2,10) |> iid(r)
+    tpr ~ Beta(10,2) |> iid(r)
+
+    pos_rate = hcat(fpr, tpr)
+
+    y ~ For(r,s) do i,j
+            Bernoulli(pos_rate[i, bad[j] + 1])
+        end
+end;
+
+using Random
+Random.seed!(1)
+
+truth = rand(m(r=5, s=14))
+
+y = NamedArray(Int.(truth.y))
+setdimnames!(y, "Reviewer",1)
+setdimnames!(y, "Sample", 2)
+display(y)
+```
+
+Here we have 20 samples, each being "reviewed" by 5 independent reviewers, with a `1` indicating a reviewer rejecting a given sample as defective. Our goal is to estimate the probability of defectiveness for each sample.
+
+On the surface, this is simple enough, and it might seem that all we really need is the counts for each column. Unfortunately, that's not quite right. 
+
+Each review has some _false positive_ and _true positive_ rate,
+
+$$
+\begin{aligned}
+\text{FPR} &= P(\text{rejected } | \text{ not defective}) \\
+\text{TPR} &= P(\text{rejected } | \text{ defective}) 
```

Comment:
The LaTeX isn't working. Maybe get rid of math mode and use "\begin{align}....\end{align}"?

Reply:
Yeah, it's not well-supported in Github (kind of a pet peeve of mine). It's arbitrary, because they render LaTeX from Jupyter notebooks just fine. Eventually we should have a copy of this in a notebook. But Github isn't the main target for this one anyway

Reply:
Gotcha